### ISSUE TITLE
Typescript Playground is not usable on Firefox Mobile

### ISSUE THREAD

**Issue by tremby** [microsoft/TypeScript-Website#2883]
- There's no way to select all in the code editor, so I'm forced to hold the backspace key to remove the default code.
- There's no way I can find to select a block of text (I would usually do this by tapping and holding, then dragging handles).
- After typing something, backspace sometimes seems to do nothing.
- If I expand the right side to see the output or errors, there's no way I can find to go back to the code again.

I'm on Android.

**Comment by Evertt** [comment]
Yes, same here on iOS. Would appreciate if this could get fixed.

**Comment by filipesabella** [comment]
Same problems here.

**Comment by d07RiV** [comment]
I don't know chrome works differently from firefox but I was able to work around some of these issues.

You can select code by tapping on a line number. Then you can drag the selection handle down to select more lines. It interferes with scrolling but is still somewhat useable. No way to select a part of a line though.

Right panel still gets stuck but you can simply refresh the page.
